app/views/clock/timezones.html.erb

```diff
@@ -1,10 +1,16 @@
-<% @timezones.each do |timezone| %>
-  <%= render 'shared/timezone', timezone: timezone %>  
-  
-  <% if user_signed_in? %>
-    <%= button_to 'Save as favorite', user_clock_index_path(current_user), method: :post, params: { name: timezone.name} %>
+<div class="row">
+  <% @timezones.each do |timezone| %>
+    <div class="col-4">
+      <div class="card text-center box-shadow" style="border-bottom: 25px solid <%= user_signed_in? ? ClockDecorator.new(timezone[:weather]).select_color : '#F0F0F0' %>;">
+      <div class="card-body">
+        <%= render 'timezone', timezone: timezone[:timezone] %>  
+        <% if user_signed_in? %>
+          <%= render 'weather', weather: timezone[:weather], timezone_name: timezone[:timezone].name %>  
+          <%= button_to 'Save as favorite', clock_index_path, method: :post, params: { name: timezone[:timezone].name}, class: "btn btn-primary btn-sm mt-2" %>
```

Comment:
I think you could delegate this responsability of capturing values (weather and timezone_name) other than the view, it would be nice just to implement something like:
```suggestion
          <%= render 'weather', weather: timezone.weather, timezone_name: timezone.name %>  
          <%= button_to 'Save as favorite', clock_index_path, method: :post, params: { name: timezone.name}, class: "btn btn-primary btn-sm mt-2" %>
```

Reply:
What do you think about a service object or a decorator?